Tasks/DownloadBuildArtifactsV1/task.json

```diff
@@ -197,6 +197,14 @@
             "groupName": "advanced",
             "required": false,
             "helpMarkDown": "Optional number of times to retry downloading a build artifact if the download fails."
+        },
+        {
+            "name": "extractTars",
```

Comment:
Considering the following pull request: https://github.com/microsoft/azure-pipelines-agent/pull/3462
I believe we also should add `cleanDestinationFolder` input.

https://github.com/microsoft/azure-pipelines-tasks/blob/71f8ac83bdd6a4838313fc54c32d5f2fb17fc2b7/Tasks/DownloadBuildArtifactsV0/task.json#L173-L179

Reply:
Fixed thanks!